From efa369cbd40df78d2dcb913d93ee4d9c0c5518dd Mon Sep 17 00:00:00 2001
From: Gabriele M <moto.falcon.git@gmail.com>
Date: Sat, 18 Feb 2017 17:17:08 +0100
Subject: [PATCH 077/173] Allow to re-open Snap from recent menu

An <activity> with an <activity-alias> doesn't automatically set
"exported" to "true" if only <activity-alias> has intent filters.
As consequence, it's impossible to re-start <activity> from the
recent applications menu. Explicitly set exported to true to fix this.

Change-Id: Id9612d704e71b5fa8093462e893a964753d67d50
---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 4afd2367a..acb5abdcf 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -45,6 +45,7 @@
             android:name="com.android.camera.CameraActivity"
             android:clearTaskOnLaunch="true"
             android:configChanges="orientation|screenSize|keyboardHidden"
+            android:exported="true"
             android:label="@string/app_name"
             android:launchMode="singleTask"
             android:screenOrientation="portrait"
-- 
2.17.1

